manifests/globals.pp

```diff
@@ -15,13 +15,20 @@
 #   The mode specifies the specifics in paths for the various RedHat SCL environments so that the module is configured
 #   correctly on their pathnames.
 #
+# [*flavor*]
+#   Flavor of PHP, either 'community' or 'zend'.
+#
+# [*zend_creds*]
+#   Hash of ZendPHP repo credentials; {username => '<USERNAME>', password => '<PASSWORD>'}
+#
```

Comment:
```suggestion
# @param flavor
#   Flavor of PHP.
#
# @param zend_creds
#   Hash of ZendPHP repo credentials; {username => '<USERNAME>', password => '<PASSWORD>'}
#
```

Maybe we should start using Puppet Strings format, even though the rest of the docs are still in the old format.